packages/events/src/storage/postgres/events/locations.ts

```diff
@@ -10,32 +10,43 @@
  */
 
 import { sql } from 'kysely'
+import { chunk } from 'lodash'
+import { logger } from '@opencrvs/commons'
 import { getClient } from '@events/storage/postgres/events'
 import { Locations, NewLocations } from './schema/app/Locations'
 
+const INSERT_MAX_CHUNK_SIZE = 10000
+
 export async function setLocations(locations: NewLocations[]) {
```

Comment:
```suggestion
export async function insertLocations(locations: NewLocations[]) {
```
if we go with this maybe do this for clarity